Tests/TokamakTests/ReconcilerStressTests.swift

```diff
@@ -40,20 +40,9 @@ final class ReconcilerStressTests: XCTestCase {
     let renderer = TestRenderer(SpookyHanger())
     let root = renderer.rootTarget
 
-    return
```

Comment:
This test was invalid, but the `return` statement made this undiscoverable. Cleaning that up and simplifying the test to actually pass.